roles/confluent.test/molecule/custom-user-plaintext-rhel/verify.yml

```diff
@@ -0,0 +1,189 @@
+---
+- name: Verify - Zookeeper User Change and custom logging path
+  hosts: zookeeper1-kafka-broker1
+  gather_facts: false
+  tasks:
+    - name: Set Zookeeper user to new user
+      set_fact:
+        zookeeper_user: zk-test-user
+
+    - name: Create Zookeeper User
+      user:
+        name: "{{zookeeper_user}}"
+        comment: "Zookeeper User"
+        system: true
+        group: cp-test-group
+
+    - name: Create Zookeeper Log Directory
+      file:
+        path: /kafka/logs/
+        state: directory
+        recurse: true
+        group: cp-test-group
+        owner: "{{zookeeper_user}}"
+        mode: 0770
+
+    - name: Restart Service
+      systemd:
+        daemon_reload: true
+        name: "{{zookeeper_service_name}}"
+        state: restarted
+
+    - import_role:
+        name: confluent.zookeeper
+        tasks_from: health_check.yml
+
+    - name: Restart Service
+      systemd:
+        daemon_reload: true
+        name: "{{kafka_broker_service_name}}"
+        state: restarted
+
+    - name: Wait for Under Replicated Partitions on Broker
+      include_tasks: ../../../../tasks/wait_for_urp.yml
+    
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: zookeeper-server.log
+        custom_path: /kafka/logs/
+        group: cp-test-group
+        full_user: zk-test-user
+          
+- name: Verify - Schema Registry custom user and custom group and custom logging directory and log file presence 
+  hosts: schema_registry
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_user.yml
+      vars:
+        user: sr-cust+
+        expected_value: sr-cust+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: sr-custom
+        group: sr-custom-group
+    
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: schema-registry.log
+        custom_path: /kafka/logs/schemaregistry/
+        group: sr-custom-group
+        full_user: sr-custom
+        
+- name: Verify - Kafka Connect custom user and custom group and custom logging directory and log file presence
+  hosts: kafka_connect
+  gather_facts: no
+  tasks:
+    - import_role:
+        name: confluent.test
+        tasks_from: check_user.yml
+      vars:
+        user: connect+
+        expected_value: connect+
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_group.yml
+      vars:
+        full_user: connect-custom
+        group: connect-custom-group
+
+    - import_role:
+        name: confluent.test
+        tasks_from: check_ownership.yml
+      vars:
+        file_name: connect-distributed.log
+        custom_path: /kafka/logs/kafkaconnect/
+        group: connect-custom-group
+        full_user: connect-custom
```

Comment:
What does the full in full_user represent? I think "user" would be simpler/clearer

Reply:
same goes for check_group.yml